.gitignore

```diff
@@ -17,6 +17,7 @@ ui/lcov.info
 
 # Data
 *.csv
+api/data/
```

Comment:
This is where all the PDFs used for testing and development will be downloaded to. I guess we might want to rename it to `pdf-data` or something since it's now unclear what kind of data it might contain (back when ombpdf was its own repo, it was relatively obvious).

Reply:
We might also want to add this directory as a cache directory for CircleCI, if we want to speed up tests.